<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cpuwindow</class>
 <widget class="QMainWindow" name="cpuwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayou_main">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QChartView" name="cCpuChart" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>200</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>320</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>320</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnBackButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="button.qrc">
           <normaloff>:/backbutton/backbutton.png</normaloff>
           <normalon>:/backbutton/backbutton.png</normalon>:/backbutton/backbutton.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>Total Usage</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbTotalCpuUsage">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 7pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>Load Avg</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbLoadAvg">
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 7pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>Temperature</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbTemperature">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 7pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="button.qrc"/>
 </resources>
 <connections/>
</ui>
